src/js/dp/feedback-form.js

```diff
@@ -0,0 +1,122 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const pageURL = window.location.href;
+    const feedbackURL = "/feedback";
```

Comment:
The `consts` should be set within your guard clause 
ie. 
```suggestion
    if (feedbackFormContainer) {
    const pageURL = window.location.href;
    const feedbackURL = "/feedback";
```